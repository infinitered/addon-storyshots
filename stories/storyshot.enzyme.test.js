import path from 'path';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import initStoryshots from '../src';

initStoryshots({
  framework: 'react',
  configPath: path.join(__dirname, '..', '.storybook'),
  renderer: mount,
  serializer: toJSON,
});
