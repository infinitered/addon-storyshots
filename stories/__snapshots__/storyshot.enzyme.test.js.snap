// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Another Button with some emoji 1`] = `
<Button
  onClick={[Function]}
>
  <Component
    className="css-1yjiefr"
    onClick={[Function]}
  >
    <button
      className="css-1yjiefr"
      onClick={[Function]}
    >
      <span
        aria-label="so cool"
        role="img"
      >
        üòÄ üòé üëç üíØ
      </span>
    </button>
  </Component>
</Button>
`;

exports[`Storyshots Another Button with text 1`] = `
<Button
  onClick={[Function]}
>
  <Component
    className="css-1yjiefr"
    onClick={[Function]}
  >
    <button
      className="css-1yjiefr"
      onClick={[Function]}
    >
      Hello Button
    </button>
  </Component>
</Button>
`;

exports[`Storyshots Button with some emoji 1`] = `
<Button
  onClick={[Function]}
>
  <Component
    className="css-1yjiefr"
    onClick={[Function]}
  >
    <button
      className="css-1yjiefr"
      onClick={[Function]}
    >
      <span
        aria-label="so cool"
        role="img"
      >
        üòÄ üòé üëç üíØ
      </span>
    </button>
  </Component>
</Button>
`;

exports[`Storyshots Button with text 1`] = `
<Button
  onClick={[Function]}
>
  <Component
    className="css-1yjiefr"
    onClick={[Function]}
  >
    <button
      className="css-1yjiefr"
      onClick={[Function]}
    >
      Hello Button
    </button>
  </Component>
</Button>
`;

exports[`Storyshots Welcome to Storybook 1`] = `
<Welcome
  showApp={[Function]}
  showKind={null}
  showStory={null}
>
  <glamorous(article)>
    <article
      className="css-1fqbdip"
    >
      <glamorous(h1)>
        <h1
          className="css-nil"
        >
          Welcome to storybook
        </h1>
      </glamorous(h1)>
      <p>
        This is a UI component dev environment for your app.
      </p>
      <p>
        We've added some basic stories inside the 
        <glamorous(code)>
          <code
            className="css-mteq83"
          >
            src/stories
          </code>
        </glamorous(code)>
         directory.
        <br />
        A story is a single state of one or more UI components. You can have as many stories as you want.
        <br />
        (Basically a story is like a visual test case.)
      </p>
      <p>
        See these sample
         
        <glamorous(glamorous(glamorous(a)))
          onClick={[Function]}
        >
          <button
            className="css-1opliz7"
            onClick={[Function]}
          >
            stories
          </button>
        </glamorous(glamorous(glamorous(a)))>
         
        for a component called 
        <glamorous(code)>
          <code
            className="css-mteq83"
          >
            Button
          </code>
        </glamorous(code)>
        .
      </p>
      <p>
        Just like that, you can add your own components as stories.
        <br />
        You can also edit those components and see changes right away.
        <br />
        (Try editing the 
        <glamorous(code)>
          <code
            className="css-mteq83"
          >
            Button
          </code>
        </glamorous(code)>
         stories located at
         
        <glamorous(code)>
          <code
            className="css-mteq83"
          >
            src/stories/index.js
          </code>
        </glamorous(code)>
        .)
      </p>
      <p>
        Usually we create stories with smaller UI components in the app.
        <br />
        Have a look at the
         
        <glamorous(a)
          href="https://storybook.js.org/basics/writing-stories"
          rel="noopener noreferrer"
          target="_blank"
        >
          <a
            className="css-ca0824"
            href="https://storybook.js.org/basics/writing-stories"
            rel="noopener noreferrer"
            target="_blank"
          >
            Writing Stories
          </a>
        </glamorous(a)>
         
        section in our documentation.
      </p>
      <glamorous(p)>
        <p
          className="css-bwdon3"
        >
          <b>
            NOTE:
          </b>
          <br />
          Have a look at the 
          <glamorous(code)>
            <code
              className="css-mteq83"
            >
              .storybook/webpack.config.js
            </code>
          </glamorous(code)>
           to add webpack loaders and plugins you are using in this project.
        </p>
      </glamorous(p)>
    </article>
  </glamorous(article)>
</Welcome>
`;
